app/Transformers/ScoreTransformer.php

```diff
@@ -184,6 +213,47 @@ public function includeMatch(LegacyMatch\Score $score)
         ]);
     }
 
+    public function includePosition(MultiplayerScoreLink $scoreLink)
+    {
+        return $this->primitive($scoreLink->position());
+    }
+
+    public function includeScoresAround(MultiplayerScoreLink $scoreLink)
```

Comment:
I'd consider moving the query logic to the model so it doesn't require needing the transformer

Reply:
I mean so `scoresAround` can be checked without having to feed the model into a transformer 👀

Reply:
there isn't much in term of logic here though. At most just moving the two `where`s over I think? Unless you mean moving the whole json response generation to the model.

Reply:
just the query without the json output